docs/release-checklist.md

```diff
@@ -0,0 +1,60 @@
+Martha Release Checklist
+=========
+
+## Prerequisites
+1. Write access to the [Martha github repo](https://github.com/broadinstitute/martha)
+1. Access to the [Quay Marthay repository](https://quay.io/repository/broadinstitute/martha)
+1. The ability to run jobs on [this Jenkins server](https://fc-jenkins.dsp-techops.broadinstitute.org)
+1. Valid [eRA Commons credentials](https://public.era.nih.gov/commons/public/login.do?TARGET=https%3A%2F%2Fpublic.era.nih.gov%2Fcommons%2FcommonsInit.do)
+
+## Steps
+
+1. Create a git tag for the new version of Martha to be deployed and push it to the github repo (if done locally).
+    - e.g.: `git tag -a v1.4 -m "my version 1.4"`
+    - [tags for past releases](https://github.com/broadinstitute/martha/releases) to figure out what it last was
+    - [compare the latest code with the last version](https://github.com/broadinstitute/martha/compare) to figure out what it should be using [semantic versioning](https://semver.org/)
```

Comment:
Replace "it":
- "to figure out what the last tag was"
- "to figure out what the next tag should be using"

Reply:
good suggestion — done!